apps/designer/app/designer/features/style-panel/sections/spacing/spacing.tsx

```diff
@@ -62,6 +62,7 @@ const styles = {
     display: "block",
     background: "transparent",
     color: "$hiContrast",
+    zIndex: 99,
```

Comment:
I fixed this same weird problem on main with zIndex: 0, I haven't fully understood why it was a problem in the first place though, its weird, there was nothing that indicates the need for z-index